articles/quickstart/webapp/rails/01-login.md

```diff
@@ -92,13 +110,13 @@ rails generate controller home show --skip-assets
 
 Add the following to the generated `show.html.erb` file:
 
-```html
+```
```

Comment:
Is there any other syntax highlighting language we could use?

Reply:
There isn't an `erb` type but the deployment looks OK:

<img width="590" alt="Screenshot 2019-08-06 15 42 18" src="https://user-images.githubusercontent.com/855223/62582304-d0257e00-b860-11e9-81f4-373d7bbe5a72.png">